pyteal/ast/router_test.py

```diff
@@ -801,3 +801,24 @@ def clear_state_method():
             assemble_helper(actual_clear_state_with_method)
             == expected_clear_state_with_method
         )
+
+
+def test_override_names():
```

Comment:
@barnjamin Via https://github.com/algorand/pyteal/pull/556, I suggest a way to strengthen the test.  Can you review to see if you feel it's worth pulling in?

Reply:
Resolving - Thanks for bringing in the change.